src/client/datascience/kernel-launcher/localKernelFinder.ts

```diff
@@ -125,10 +125,6 @@ export class LocalKernelFinder implements ILocalKernelFinder {
         }
     }
 
-    public async listNonPythonKernels(cancelToken?: CancellationToken): Promise<LocalKernelConnectionMetadata[]> {
-        return this.filterKernels(await this.nonPythonKernelFinder.listKernelSpecs(false, cancelToken));
```

Comment:
this same method is called inside `this.listKernlesWithoutCache` which is inturn called in `listKernels`